### ISSUE TITLE
Make the background mountains tile, and cover the bottom of the screen

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/godot-platformer-2d#172]
The mountains in the background should repeat seamlessly and cover the bottom of the sky, regardless of the level's size.

![Screenshot from 2020-01-16 19-29-03-fs8](https://user-images.githubusercontent.com/12694995/72552138-8375b280-3896-11ea-9d25-2a1944c6f2c8.png)

Right now, the mountains' width is set by hand.

The mountains should have some parallax, at least horizontally.

Here's a sample script I wrote for Flossy Gnu that repeats a sprite/texture node infinitely, for endless runner games. You're free to solve the problem in another way if you have a more efficient approach.

```gdscript
tool
extends Node2D
# Repeats a sprite horizontally infinitely by spawning enough sprites to always
# fill the view

onready var spawn_position = $SpawnPosition

export(PackedScene) var sprite_scene

var WINDOW_WIDTH = ProjectSettings.get('display/window/size/width')
var sprite_width = 0.0


func _ready():
	if Engine.editor_hint:
		spawn_position.add_child(sprite_scene.instance())
	else:
		var sprite = sprite_scene.instance()
		sprite_width = sprite.texture.get_width() * sprite.scale.x
		var amount_to_fill_screen = ceil(WINDOW_WIDTH / sprite_width)
		for i in range(amount_to_fill_screen + 2):
			spawn_sprite()


# Offsets the spawner node and spawns a new sprite
func spawn_sprite():
	var sprite = sprite_scene.instance()
	sprite.connect("tree_exited", self, "spawn_sprite")
	sprite.position = spawn_position.position
	call_deferred("add_child", sprite) # to avoid error on reload
	spawn_position.position.x += sprite_width
```

**Comment by henriiquecampos** [comment]
Looks like something a ParallaxLayer could fix. The gradient on the Sky will be tricky tho, since it shouldn't simply repeat. I'll try to come out with something. Do we have the Clouds sprites separate from the Sky?

**Comment by NathanLovato** [comment]
The sky in the background doesn't have to scroll, it's just a vertical background. A static texture will do

**Comment by henriiquecampos** [comment]
I think this is closed in 89c2fe58024db55886612a5884c3582c61671ea4
